src/ja/server/database/database.py

```diff
@@ -20,15 +20,6 @@ def find_job_by_id(self, job_id: str) -> DatabaseJobEntry:
         @return The stored data about the job, or None if no such job was found.
         """
 
-    @abstractmethod
```

Comment:
Why did you remove `find_job_by_label`, and then you reimplement it using `query_jobs` and manually filter through them?

Reply:
Because I couldn't make it work with `session.query(Job).filter(Job.label == label)`. The method didn't work.

Reply:
I opened a PR that fixes the method: https://github.com/DistributedTaskScheduling/JobAdder/pull/135

Reply:
That PR is now merged, so it makes the most sense to use the database function.